install

```diff
@@ -55,3 +55,4 @@ echo "alias rosinit='source /opt/ros/indigo/setup.bash'" >> ~/.bashrc
 echo "alias rosdevel='source ~/catkin_ws/devel/setup.bash' # needed for autocompletion" >> ~/.bashrc
 echo "rosinit     # put these in your .bashrc too so they run everytime" >> ~/.bashrc
 echo "rosdevel    # you open a new terminal window/tab." >> ~/.bashrc
+echo "source ~/catkin_ws/src/buzzmobile/bin/activate # activate venv" >> ~/.bashrc
```

Comment:
This is bad, please make this an alias at the very least, you don't want to always source the script by default.